testdata/values.txt

```diff
@@ -0,0 +1,30 @@
+mito -use collections,try src.cel
+! stderr .
+cmp stdout want.txt
+
+-- src.cel --
+{
+	"good_instance": {"a": ["aa", "ab"], "b": [2, 3], "c": {"cc": 4, "cd": 5}}.values(),
```

Comment:
Let's have this out of order.

Reply:
working after new commit with modified sorting logic using struct.